catshadow/Makefile

```diff
@@ -3,7 +3,7 @@ ldflags="-X github.com/katzenpost/katzenpost/core/epochtime.WarpedEpoch=${warped
 uid=$(shell [ "$$SUDO_UID" != "" ] && echo "$$SUDO_UID" || id -u)
 gid=$(shell [ "$$SUDO_GID" != "" ] && echo "$$SUDO_GID" || id -g)
 docker_user=${uid}:${gid}
-docker_args=--user ${docker_user} -v $(shell readlink -f ..):/go/katzenpost -e GOCACHE=/tmp/gocache --network=host --ulimit memlock=1048576 --rm -it --volume /usr/local/include/sphincsplus:/usr/local/include/sphincsplus
+docker_args=--user ${docker_user} -v $(shell readlink -f ..):/go/katzenpost -e GOCACHE=/tmp/gocache --network=host --ulimit memlock=1048576 --rm --volume /usr/local/include/sphincsplus:/usr/local/include/sphincsplus
```

Comment:
You are using an out of date version of this Makefile.... the sphincsplus stuff is no longer needed here as per recent changes in main branch.

Reply:
Thank you for notifying me that upstream has changed.